networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -215,12 +249,18 @@ where
 				anyhow::bail!("Invalid DA height: {:?}", da_height);
 			}
 
+			// decompress the block bytes
+			let block = tokio::task::spawn_blocking(move || {
+				let decompressed_block_bytes = zstd::decode_all(&block_bytes[..])?;
```

Comment:
It seems I was mistaken about the format freeze and we get to change the blob's inner format without resetting the chain. Will this really work seamlessly?

Reply:
Like I said last week, I was looking into the ramifications and this is indeed fine if we prune the DA. It means that our partners need a trust start state, but they were going to need one anyone.